packages/components/src/Card/Card.stories.mdx

```diff
@@ -68,6 +92,105 @@ Card titles should be sentence-cased.
 
 ## Variations
 
+### Header with action
+
+Pass a Button in as the header action to present the user with an action related
+to the Card contents.
+
+<Canvas>
+  <Card
+    header={{
+      title: "Services",
+      action: (
```

Comment:
I keep trying to pass this component in without the parentheses but it auto-adds them on save? :help: